boards/chipyard/base-workloads/br-base.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name" : "br-base",
```

Comment:
Identically named workloads result in identically named output images (e.g., `br-base-bin`).
In a future change, it might be a good idea to prefix the filenames with the board name to allow the images to coexist.